src/Cms/Media.php

```diff
@@ -97,16 +97,17 @@ public static function thumb($model, string $hash, string $filename)
 	{
 		$kirby = App::instance();
 
-		// assets
-		if (is_string($model) === true) {
-			$root = $kirby->root('media') . '/assets/' . $model . '/' . $hash;
-		// parent files for file model that already included hash
-		} elseif ($model instanceof File) {
-			$root = dirname($model->mediaRoot());
-		// model files
-		} else {
-			$root = $model->mediaRoot() . '/' . $hash;
-		}
+		$root = match (true) {
+			// assets
+			is_string($model)
+				=> $kirby->root('media') . '/assets/' . $model . '/' . $hash,
+			// parent files for file model that already included hash
+			$model instanceof File
+				=> dirname($model->mediaRoot()),
+			// model files
+			default
+			=> $model->mediaRoot() . '/' . $hash
+		};
```

Comment:
Same here.

Reply:
I guess it's really a matter of taste, but I'm also on @distantnative's side here.